### PR TITLE
CB-17836 - Roll forward sync fails after upgrade because of cloudera-…

### PR DESCRIPTION
**Pull request by pdarvasi**
…manager-server version consistency validation

This commit:
 - removes CM server and agent consistency check as there might be
different server versions in the CM Server and the other nodes
 - introduces parameter for multiple version consistency check for the
same reason

See detailed description in the commit message.

### CODE REVIEW

**Review by nadamdb — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/PackageName.java [hunk 1]
```diff
@@ -21,6 +23,14 @@ public void setPattern(String pattern) {
...
+    public boolean getValidateForMultipleVersions() {
+        return validateForMultipleVersions;
+    }
+
+    public void setValidateForMultipleVersions(boolean validateForMultipleVersions) {
+        this.validateForMultipleVersions = validateForMultipleVersions;
+    }
+
     @Override
     public String toString() {
```
[the remark this answers is not in the document]
Comment (pdarvasi) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/PackageName.java:35]: I've left it out on purpose as it is quite an implementation detail, do you insist on putting it there?

[the remark this answers is not in the document]
Comment (nadamdb) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/PackageName.java:35]: If it was intentional it's fine by me

**Review by nadamdb — APPROVED** [review]
LGTM

**Comment by lnardai** [comment]
Change is unrelated to the test failure, going to merge this.

Failed Tests: 1 (+1)
mock-tests / newway mock testcases
[com.sequenceiq.it.cloudbreak.testcase.mock.EnvironmentStartStopTest.testCreateStopStartEnvironment](http://ci-cloudbreak.eng.hortonworks.com/job/cloudbreak-pull-request-builder-integration-test/26424/testngreports/com.sequenceiq.it.cloudbreak.testcase.mock/EnvironmentStartStopTest/testCreateStopStartEnvironment)

**Pull request merged**